tests/functional/vault/test_withdrawal.py

```diff
@@ -259,3 +259,25 @@ def test_withdrawal_with_reentrancy(
     # given previous setup the withdraw should revert from reentrancy guard
     with brownie.reverts():
         vault.withdraw(vault.balanceOf(gov), {"from": gov})
+
+
+def test_token_transfer_fail(token, gov, Vault):
```

Comment:
Not necessary if merged into previous test

Reply:
Understood!